assets/js/components/event/HlsPlayer.tsx

```diff
@@ -17,6 +19,9 @@ const HlsPlayer = ({ hlsUrl, presenterName }: HlsPlayerProps) => {
           <div className="HlsTopBar">
             <div className="HlsPresenterName">{presenterName}</div>
           </div>
+          <div className="HlsBottomBar">
+            <AnimationComponent eventChannel={eventChannel} />
```

Comment:
I was wondering if we could move the reaction button to here and make it some kind of icon, similarly to the demo mock UI.

Reply:
We could, but I am not sure how this will compose with the menu of hls-player. I think we could add this when we will change the stream player. By the way for this future task, we could use this:
https://github.com/muxinc/media-chrome